config/routes.rb

```diff
@@ -13,15 +13,11 @@ def call(env)
 
 Rails.application.routes.draw do
 
-  constraints MasterDomainConstraint do
+  constraints PortConstraint.new(ENV.fetch('PROMETHEUS_EXPORTER_PORT', 9090).to_i) do
```

Comment:
Won't this conflict with https://github.com/3scale/porta/blob/f46f350ed0e2bb5589f6aec577f5fb79e9f91d50/config/initializers/sidekiq.rb#L36-L38?

Reply:
No because sidekiq and unicorn do not run in the same host.
We could change `SIDEKIQ_PROMETHEUS_EXPORTER_PORT` if you think it is necessary.

Anyway we want also to normalize all prometheus exporter.
Only one per process using default port 9090

This is not achievable for now as sidekiq-prometheus-exporter gem overrides the content of previous rack application.
So we cannot chain exporters https://github.com/Strech/sidekiq-prometheus-exporter/blob/15879ca30830a94ed254a443448859c84fb4e88c/lib/sidekiq/prometheus/exporter.rb#L47-L51

Follow https://issues.redhat.com/browse/THREESCALE-2468 for more information